apps/site/assets/package.json

```diff
@@ -132,6 +133,7 @@
     "stylelint": "stylelint css/**/*.scss --ignore-path .stylelintignore",
     "prettier": "prettier --write \"{js,ts}/**/*.{js,ts,tsx}\"",
     "prettier:check": "npm run prettier -- --list-different",
+    "prepare": "cd ../../.. && husky install .husky",
```

Comment:
question: Would it work to move this to the root `package.json` file and not need the `cd` bit?

Reply:
to this question, might need some prefixing hackery e.g. `"$(npm bin --prefix apps/site/assets)/husky install .husky` might work

Reply:
Could husky instead just be installed at the root of the project? Or would you not be able to use it from the `site` package scripts? If that's the case seems like it might be fine as is.

Reply:
The issue with installing it in the root is we have no other modules installed there.  We could, definitely do it, just setting new precedents.